ams/server/src/test/java/com/netease/arctic/server/catalog/TestServerCatalog.java

```diff
@@ -18,73 +18,134 @@
 
 package com.netease.arctic.server.catalog;
 
+import com.netease.arctic.AmoroCatalog;
+import com.netease.arctic.ams.api.properties.CatalogMetaProperties;
 import com.netease.arctic.formats.AmoroCatalogTestHelper;
 import com.netease.arctic.formats.IcebergHadoopCatalogTestHelper;
+import com.netease.arctic.formats.MixedIcebergHadoopCatalogTestHelper;
 import com.netease.arctic.formats.PaimonHadoopCatalogTestHelper;
 import com.netease.arctic.hive.formats.IcebergHiveCatalogTestHelper;
+import com.netease.arctic.hive.formats.MixedIcebergHiveCatalogTestHelper;
 import com.netease.arctic.hive.formats.PaimonHiveCatalogTestHelper;
+import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.Parameterized;
 
+import java.util.HashMap;
+
 @RunWith(Parameterized.class)
 public class TestServerCatalog extends TableCatalogTestBase {
 
   private final String testDatabaseName = "test_database";
 
   private final String testTableName = "test_table";
 
+  private final String testTableNameFilter = "test_table_filter";
+
   public TestServerCatalog(AmoroCatalogTestHelper<?> amoroCatalogTestHelper) {
     super(amoroCatalogTestHelper);
   }
 
   @Parameterized.Parameters(name = "{0}")
   public static Object[] parameters() {
     return new Object[] {
-      PaimonHadoopCatalogTestHelper.defaultHelper(),
-      PaimonHiveCatalogTestHelper.defaultHelper(),
-      IcebergHadoopCatalogTestHelper.defaultHelper(),
-      IcebergHiveCatalogTestHelper.defaultHelper()
+      new PaimonHadoopCatalogTestHelper(
```

Comment:
The changes will affect all test cases. I suggest not testing in this way. How about add a new test case like 
```java
  @Test
  public void listTablesWithTableFilter() {
    // 1.create some databases and tables
    // 2.check the tables without filter
    // 3.set table filter
    getExternalCatalog().updateMetadata(); 
    // 4.check the table list with filter
    // 5.finally unset the table filter, remove the databases and tables in this test case
  }

```

Reply:
done.